EpikCord/ext/tasks.py

```diff
@@ -0,0 +1,102 @@
+import datetime
+from typing import Callable, Optional
+from time import time as _time
+import asyncio
+from ..exceptions import TaskFailedError
+
+_EPOCH = datetime.datetime(1970, 1, 1)
+
+
+class TimeParser:
+    @staticmethod
+    def get_time_since_epoch(
+        year: int, month: int, day: int, hour: int, minute: int, seconds: float
+    ) -> float:
+        date = datetime.datetime(year, month, day, hour, minute, seconds)
+
+        delta = date - _EPOCH
+        return delta.total_seconds()
+
+    @staticmethod
+    def total_seconds(
+        year: int = 0,
+        month: int = 0,
+        day: int = 0,
+        hour: int = 0,
+        minute: int = 0,
+        seconds: float = 0,
+    ) -> float:
+        month += year * 365
+        day += month * 30
+        hour += day * 24
+        minute += hour * 60
+        seconds += minute * 60
+        return seconds
+
+
+class Tasks:
+    def __init__(self, client) -> None:
+        self.client = client
+
+    def add_task(
+        self,
+        task: Callable[..., None],
+        interval: Optional[int] = None,
+        start: Optional[float] = None,
+        until: Optional[float] = None,
+        **kwargs,
+    ):
```

Comment:
Kwargs for max execution? 

Like
```py
limit = 5
```
So the task run 5 times exactly.

Reply:
we have a kwarg for that. "instances" but if you want i can rename it to limit

Reply:
"instances" is too vague, so i'd recommend a change.

Reply:
ok

Reply:
> we have a kwarg for that. "instances" but if you want i can rename it to limit

Well it's isn't specified by default, meaning any completion tools would not suggest it.